src/requirements/reqs.json

```diff
@@ -2137,5 +2137,31 @@
                 }
             ]
         }
-    }
+    },
```

Comment:
Make sure not to include this in this pull request since it is not related to the 4040 page. Could you remove the changes in this file?